pkg/proxy/iptables/proxier.go

```diff
@@ -667,14 +667,14 @@ func (proxier *Proxier) deleteEndpointConnections(connectionMap []proxy.ServiceE
 const endpointChainsNumberThreshold = 1000
 
 // Assumes proxier.mu is held.
-func (proxier *Proxier) appendServiceCommentLocked(args []string, svcName string) {
+func (proxier *Proxier) appendServiceCommentLocked(args []string, svcName string) []string {
 	// Not printing these comments, can reduce size of iptables (in case of large
 	// number of endpoints) even by 40%+. So if total number of endpoint chains
 	// is large enough, we simply drop those comments.
 	if proxier.endpointChainsNumber > endpointChainsNumberThreshold {
-		return
+		return args
```

Comment:
The comment above suggests that not returning the args was an explicit decision. I'm not sure that re-adding them is the right choice.

Reply:
We need `return args` here, otherwise on L1187 in this file will replace `args` by nil - see https://github.com/kubernetes/kubernetes/pull/81886#discussion_r328288746.

Reply:
This is fixed in #81563

Reply:
Should we wait for that PR to merge?